qase-pytest/requirements.txt

```diff
@@ -1,17 +1,26 @@
-# =============================================================================
-# DEPRECATION WARNING:
-#
-# The file `requirements.txt` does not influence the package dependencies and
-# will not be automatically created in the next version of PyScaffold (v4.x).
-#
-# Please have look at the docs for better alternatives
-# (`Dependency Management` section).
-# =============================================================================
-#
-# Add your pinned requirements so that they can be easily installed with:
-# pip install -r requirements.txt
-# Remember to also add them in setup.cfg but unpinned.
-# Example:
-# numpy==1.13.3
-# scipy==1.0
-#
+apitist==4.3.0
+attrs==21.4.0
+cattrs==1.1.2
+certifi==2022.6.15
+charset-normalizer==2.1.0
+convclasses==2.0.0
+coverage==6.4.2
+filelock==3.7.1
+idna==3.3
+iniconfig==1.1.1
+install==1.3.5
+packaging==21.3
+pendulum==2.1.2
+pluggy==1.0.0
+py==1.11.0
+pyparsing==3.0.9
+pytest==7.1.2
+pytest-cov==3.0.0
+python-dateutil==2.8.2
+pytzdata==2020.1
+qase-pytest==2.3.2
```

Comment:
Should we pick up a version here and a line below?

Reply:
It just pip3 freeze. btw also we can drop these 2 lines